Assets/Scripts/Drinkables/Drink Pickup/TavernEntrance.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TavernEntrance : MonoBehaviour
+{
+    [SerializeField] private float tavernCoolDown = 30;
+
+    private bool tavernOpen;
+    private float cooldown;
+
+    private void Start()
+    {
+        tavernOpen = true;
+        cooldown = 0;
+    }
+
+    private void Update()
+    {
+        if (!tavernOpen)
+        {
+            cooldown += Time.deltaTime;
+            this.transform.Find("Illumination").gameObject.SetActive(false);
```

Comment:
I would make a variable for the illumination and only call `Find` once instead of twice to boost performance